erpnext/healthcare/doctype/lab_test/lab_test_list.js

```diff
@@ -2,18 +2,18 @@
 (c) ESS 2015-16
 */
 frappe.listview_settings['Lab Test'] = {
-	add_fields: ["name", "status", "invoiced"],
-	filters:[["docstatus","=","0"]],
+	add_fields: ['name', 'status', 'invoiced'],
+	filters:[['docstatus','=','0']],
 	get_indicator: function(doc) {
-		if(doc.status=="Approved"){
-			return [__("Approved"), "green", "status,=,Approved"];
+		if(doc.status=='Approved'){
```

Comment:
```suggestion
		if (doc.status=='Approved') {
```
Add spaces for all such conditions